src/components/contacts/contacts.css

```diff
@@ -1,46 +1,47 @@
 .contact-form {
-  margin: 0 auto;
+    margin-top: 20px
```

Comment:
there were times, when all vertical margins were or "top", or "bottom". And most people used bottom. Are you sure to go against the stream? :)

Reply:
Fixed